src/util/requireOrImportConfig.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+// Node v12.17+ exposes `import` within CJS files
+// in order to `require` ESM files.
+
+// This file is intentionally excluded from `babel` (and `eslint`)
+// to avoid transpiling away the `import` statement
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+exports.default = requireOrImportConfig;
+
+function requireOrImportConfig(config) {
+  try {
+    return require(config)
+  } catch (e) {
+    if (e.code === 'ERR_REQUIRE_ESM') {
```

Comment:
What if it is a different exception? Right now you will return `null`, but I think you should throw the original exception.

Reply:
Good catch, that makes total sense. I will update this!